nimare/dataset/dataset.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+import json
+
+
+class Dataset(object):
+    """Storage container for a coordinate- and/or image-based meta-analytic
+    dataset/database.
+    """
+    def __init__(self, dataset_file):
+        with open(dataset_file, 'r') as fo:
+            self.data = json.load(fo)
+
+    def has_data(self, dat_str):
+        dat_str = dat_str.split(' AND ')
+        for ds in dat_str:
+            try:
+                self.data.get(None)
+            except:
+                raise Exception('Nope')
+
+    def get(self, search='', algorithm=None):
```

Comment:
Given the scope of the project and the variety of different data types this object is going to store, I think we may want more explicit accessor methods (e.g., `get_studies`, `get_metadata`, etc.). Fine to leave this as a placeholder, but ultimately we'll probably replace it with a bunch of more specific methods.

Reply:
I'll add in some more empty methods, just so there's a blueprint.